src/test/java/com/demoqa/DemoQaTest.java

```diff
@@ -1,10 +1,168 @@
 package com.demoqa;
 
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.interactions.Actions;
+
+import java.time.Duration;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 
 public class DemoQaTest {
+
+    private static final By ELEMENTS_CARD = By.xpath("//div[@class='card-body']/h5[text()='Elements']");
+    private static final By BUTTONS_MENU_ITEM = By.id("item-4");
+    private static final By DOUBLE_CLICK_BUTTON = By.id("doubleClickBtn");
+    private static final By MESSAGE_DOUBLE_CLICK_BUTTON = By.id("doubleClickMessage");
+    private static final String EXPECTED_MESSAGE_DOUBLE_CLICK_BUTTON = "You have done a double click";
+    private static final By RIGHT_CLICK_BUTTON = By.id("rightClickBtn");
+    private static final By MESSAGE_RIGHT_CLICK_BUTTON = By.id("rightClickMessage");
+    private static final String EXPECTED_MESSAGE_RIGHT_CLICK_BUTTON = "You have done a right click";
+    private static final By SPAN_TEXT_RADIO_BUTTON = By.xpath("//span[text()='Radio Button']");
+    public static final By LABEL_FOR_YES_RADIO = By.xpath("//label[@for='yesRadio']");
+    public static final By RESULT_CHOICE_RADIO =By.className("text-success");
+    private static final String EXPECTED_MESSAGE_YES_RADIO = "Yes";
+    private static final By LABEL_FOR_NOTES_CHECKBOX = By.xpath("//label[@for='tree-node-notes']");
+    private static final By HOME_EXPAND =By.className("rct-collapse-btn");
```

Comment:
please follow the code style
```suggestion
    private static final By HOME_EXPAND = By.className("rct-collapse-btn");
```